src/common/utils/index.js

```diff
@@ -2,7 +2,7 @@ export const idExtractor = item => `${item.id}`;
 // eslint-disable-next-line no-underscore-dangle
 export const underscoreIdExtractor = item => item._id;
 
-export const isIdForCurrentUser = (id, currentUser) => currentUser.id === id;
+export const isIdForCurrentUser = (id, currentUser) => currentUser.id == id;
```

Comment:
Is threequals (===) still the "best practice" in JS, or has that changed?  I haven't really kept up

Reply:
`===` is the standard but it only compares the same types (String -> String)
however the `id` we pass in can sometimes be an `Int` and thus we use `==` which applies type conversion automatically